src/reducer.spec.ts

```diff
@@ -121,3 +121,29 @@ test(
     ).toBeObservable(expected$);
   })
 );
+
+test(
+  'combineReducers should forward namespace',
+  marbles((m) => {
+    const action$ = m.hot('  1', actions);
+    const expected$ = m.hot('2', numbers);
+
+    const verifyNamespaceReducer = reducer(
+      actionCreators.incrementOne,
+      (_, __, namespace) => {
```

Comment:
This is an example of the unfortunate syntax. This is how we will use it for reset actions in ardoq-front.